tests/test_plugin_falcon.py

```diff
@@ -545,3 +550,40 @@ def test_falcon_optional_alias_response(client):
     )
     assert resp.status_code == 200, resp.json
     assert resp.json == {"schema": "test"}, resp.json
+
+
+@pytest.mark.skipif(not PYDANTIC2, reason="only matters if using both model types")
+def test_falcon_validate_both_v1_and_v2_validation_errors(client):
+    class CompatibilityView:
```

Comment:
Inlined this view because adding it to the test app then causes issue with the [generated spec tests](https://github.com/0b01001001/spectree/blob/main/tests/test_plugin.py#L25).